CONTRIBUTING.md

```diff
@@ -1,15 +1,16 @@
 # Contributing
 
-[fork]: /fork
-[pr]: /compare
+[fork]: https://github.com/gitgitgadget/gitgitgadget/fork
+[pr]: ../../compare
 [style]: https://palantir.github.io/tslint/
 [code-of-conduct]: CODE_OF_CONDUCT.md
+[TODO]: TODO.md
```

Comment:
> Isn't it still missing that .md extension?

I put the extension in the list of links at the beginning of the file

Reply:
Ah! Thanks for enlightening me.